packages/utilities/psammead-locales/moment/sr.js

```diff
@@ -10,8 +10,12 @@ moment.updateLocale('sr', {
   relativeTime: {
     past: 'Pre %s',
     m: '1 minuta',
-    mm: 'minuta: %d',
+    mm: '%d minuta',
     h: '1 sata',
-    hh: 'sati: %d',
+    hh: function(number) {
+      return number < 5
+        ? String(number).concat(' sata')
+        : String(number).concat(' sati');
```

Comment:
eslint didn't like my first two approaches here, which were:
`number + ' sata'`
and
`\`${number} sata`

Reply:
`${number} sata` should work.

Reply:
It did not.  Sorry github removed the ` symbols in my comment. Eslint did not want me to use that format.

Reply:
Hmm. The `moment` folder has its own linting rules that specify we can only use es5 in there. Does anyone know if this is a limitation of moment.js? But there is also a linting rule defined that says we have to use templating instead of concatenation, which is why it won't let you use either. So either we allow es6, but I'm sure it's been disabled for a reason, or we allow string concatenation.

Reply:
In case it helps, these linting rules where added in [this](https://github.com/bbc/psammead/pull/2080) PR.